src/teleop/jetson/jetson_teleop.py

```diff
@@ -2,18 +2,17 @@
 # Node for teleop-related callback functions
 
 from math import copysign
+from turtle import forward
```

Comment:
Idk how this got here but probably shouldn't be here

Reply:
bruh

Reply:
teleop moment

Reply:
do you use VSCode?

Reply:
> do you use VSCode?

Any reason why u mention this? Otherwise I will just close this comment

Reply:
I think he was asking cause VSCode will automatically add imports sometimes, something to be cautious of